openshift-ci/test/e2e/main_test.go

```diff
@@ -0,0 +1,11 @@
+package e2e
```

Comment:
I think this test directory should be in the root of the repo rather than under `openshift-ci`.

Reply:
Only just saw this. It's here for a reason, it's not a real test for the operator, it's just one for the Dockerfile itself.